app/enterprise/changelog.md

```diff
@@ -4,6 +4,154 @@ no_search: true
+- Improve support for OPTIONS requests.
+  [#7830](https://github.com/Kong/kong/pull/7830)
+
+#### Plugins
+- **New plugin:** [jq](/hub/kong-inc/jq) (`jq`)
+  The jq plugin enables arbitrary jq transformations on JSON objects included in API requests or responses.
+- [Kafka Log](/hub/kong-inc/kafka-log) (`kafka-log`)
+  The Kafka Log plugin now supports TLS, mTLS, and SASL auth.
+- [Kafka Upstream](/hub/kong-inc/kafka-upstream) (`kafka-upstream`)
+  The Kafka Upstream plugin now supports TLS, mTLS, and SASL auth.
+- [Rate Limiting Advanced](/hub/kong-inc/rate-limiting-advanced) (`rate-limiting-advanced`)
+  - The Rate Limiting Advanced plugin now has a new identifier type: `path`, which allows rate limiting by matching request paths.
+  - The plugin now has a `local` strategy in the schema. The local strategy automatically sets `config.sync_rate` to -1.
+- [OPA](/hub/kong-inc/opa) (`opa`)
+  The OPA plugin now has a request path parameter, which makes setting policies on a path easier for administrators.
+- [Canary](/hub/kong-inc/canary) (`canary`)
+  The Canary plugin now has the option to hash on the header (falls back on IP, and then random).
+- [OpenID Connect](/hub/kong-inc/openid-connect) (`openid-connect`)
+  Upgrade to v2.0.x to maintain version compatibility with older data planes.
+  - The OpenID Connect plugin can now handle JWT responses from a `userinfo` endpoint.
+  - The plugin now supports JWE Introspection.
+- [AWS-Lambda](/hub/kong-inc/aws-lambda) (`aws-lambda`) 
+  The plugin will now try to detect the AWS region by using `AWS_REGION` and
+  `AWS_DEFAULT_REGION` environment variables (when not specified with the plugin configuration).
+  [#7765](https://github.com/Kong/kong/pull/7765)
+- [Datadog](/hub/kong-inc/datadog) (`datadog`)
+  The Datadog plugin now allows `host` and `port` config options be configured from `ENV` variables.
+  [#7463](https://github.com/Kong/kong/pull/7463)
+- [Prometheus](/hub/kong-inc/prometheus) (`prometheus`) 
+  The new `data_plane_cluster_cert_expiry_timestamp` exposes DP cert expiry timestamp.
+  [#7800](https://github.com/Kong/kong/pull/7800).
+- [GRPC-Gateway](/hub/kong-inc/grpc-gateway) (grpc-gateway)
+  - The plugin can decode entities of type `.google.protobuf.Timestamp`
+    [#7538](https://github.com/Kong/kong/pull/7538)
+  - Added support for structured URI arguments
+    [#7564](https://github.com/Kong/kong/pull/7564)
+- [Request Termination](/hub/kong-inc/request-termination) (`request-termination`)
+  - New `trigger` config option, which makes the plugin activate for any requests with a header or query parameter
+    named like the trigger
+    [#6744](https://github.com/Kong/kong/pull/6744).
+  - The `request-echo` config option. If not set, the plugin activates on all requests.
+    If set, then the plugin only activates for any requests with a header or query parameter
+    named like the trigger
+    [#6744](https://github.com/Kong/kong/pull/6744).
```

Comment:
```suggestion
- New `trigger` config option, which makes the plugin only activate for any requests with a header or query parameter
  named like the trigger. This can be a great debugging aid, without impacting actual traffic being processed.
  [#6744](https://github.com/Kong/kong/pull/6744).
- The `request-echo` config option was added. If set, the plugin responds with a copy of the incoming request.
  This eases troubleshooting when Kong is behind one or more other proxies or LB's, especially when combined with
  the new 'trigger' option.
  [#6744](https://github.com/Kong/kong/pull/6744).
```